src/components/verification-grid-tile/verification-grid-tile.ts

```diff
@@ -252,24 +257,26 @@ export class VerificationGridTileComponent extends SignalWatcher(AbstractCompone
     `;
   }
 
-  private decisionColor(decisionIndices: number[]): CSSResult {
-    // if the user has only made one decision then we want to show a solid color
-    if (decisionIndices.length === 1) {
-      const solidColor = decisionIndices[0];
-      return unsafeCSS(`background-color: var(--decision-color-${solidColor});`);
-    }
-
-    // I sort the decision indices so that no matter what order the user
-    // selects the decisions, the gradient will always be the same
-    // if the indices were not ordered, tiles with the same decisions applied
-    // but in a different order would have different gradients
-    const percentageStep = 100 / decisionIndices.length;
-    const gradientColors = decisionIndices.sort().map((color: number, index: number) => {
-      const percentage = percentageStep * (index + 1);
-      return `var(--decision-color-${color}) 0 ${percentage}%`;
-    });
+  private meterSegmentsTemplate(): TemplateResult<1> {
+    const undecidedColor = "var(--oe-panel-color)";
 
-    return unsafeCSS(`background: conic-gradient(${gradientColors});`);
+    return html`
+      ${repeat(this.requiredTags, (tag: Tag) => {
+        const decision = this.model.decisions.get(tag.text);
+        const decisionText = decision ? decision.confirmed : "no decision";
```

Comment:
even if we haven't decided on how to represent skip/unsure visually it seems like we could put the text in the tooltip easily enough.

suggestion: `decision.toString()` (move formatting to decision object)

Reply:
> even if we haven't decided on how to represent skip/unsure visually it seems like we could put the text in the tooltip easily enough.

This is how it currently behaves and we got the functionality for free because _skip_ and _unsure_ decisions are in our `DecisionOptions` enum.

No special case needed 🎉

> suggestion: decision.toString() (move formatting to decision object)

Good idea, I'll see if I can do this

I'm not sure if I can move all the formatting functionality into the decision model because sometimes a tag does not have a decision model (if it is undecided)

Reply:
> suggestion: decision.toString() (move formatting to decision object)

Discussed offline

The following implementation was deemed to be sufficiant

![image](https://github.com/user-attachments/assets/53c6e3f7-0984-4fb4-98ce-d0341740a890)